backend/src/transaction_event/protocol_level_tokens.rs

```diff
@@ -641,23 +653,58 @@ impl PreparedTokenUpdate {
         .execute(tx.as_mut())
         .await?;
 
+        self.update_metrics_plt_cumulative_event_count(tx, transaction_index).await?;
+
         match self.event_type {
             TokenUpdateEventType::Mint => {
                 if let Some(ref target) = self.target {
+                    let account_exists = self.check_if_account_exists(tx, target).await?;
+
                     self.update_total_minted(tx).await?;
                     self.update_account_balance(tx, target, &self.plt_amount_change).await?;
+                    if !account_exists {
+                        self.update_metrics_plt_unique_account_count(tx, transaction_index).await?;
+                    }
                 }
             }
             TokenUpdateEventType::Burn => {
                 if let Some(ref target) = self.target {
+                    let account_exists = self.check_if_account_exists(tx, target).await?;
```

Comment:
The account would need to hold PLT to burn, so `account_exists` would always be `true` right?
We can skip this query and skip updating the unique counter